documentation/public/user-guide/rust-crate/index.md

```diff
@@ -4,4 +4,3 @@
 -   [Using the CLI](./using-the-cli.md)
 -   [Using the Parser](./using-the-parser.md)
 -   [Using the Cursor](./using-the-cursor.md)
--   [Using Queries](./using-queries.md)
```

Comment:
i think this line was unintentionally removed?

Reply:
done